interface/main/calendar/add_edit_event.php

```diff
@@ -1602,8 +1696,14 @@ function isRegularRepeat($repeat){
   <td nowrap>
 
 <?php
-generate_form_field(array('data_type'=>1,'field_id'=>'apptstatus','list_id'=>'apptstat','empty_title'=>'SKIP'), $row['pc_apptstatus']);
+if($_GET['group']!=true) {
+    generate_form_field(array('data_type' => 1, 'field_id' => 'apptstatus', 'list_id' => 'apptstat', 'empty_title' => 'SKIP'), $row['pc_apptstatus']);
+}
+else{
+    generate_form_field(array('data_type' => 1, 'field_id' => 'apptstatus', 'list_id' => 'groupstat', 'empty_title' => 'SKIP'), $row['pc_apptstatus']);
+}
```

Comment:
The group attendance list contains only part of the statuses of regular appt statuses